packages/typespec-python/test/azure/generated/azure-mgmt-spheredpg/CHANGELOG.md

```diff
@@ -1,5 +0,0 @@
-# Release History
```

Comment:
Already have ARM test case in cadl-ranch so don't need local specification of sphere.